css/styles.css

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable string-quotes */
```

Comment:
Delete this. Don't add lint rules without asking. We have rules setup for a reason